eq-author/src/components/AnswerContent/Decimal/index.test.js

```diff
@@ -24,7 +24,7 @@ describe("Decimal Property", () => {
     expect(getByTestId("number-input")).toBeTruthy();
   });
 
-  it("should render with Error", () => {
+  it("should render with a decimal inconsistency error", () => {
```

Comment:
These 3 tests could be improved by checking that the correct error message is being displayed, in addition to the input field being highlighted